Remove "one line summary" from pr template (#1073)
ManimCommunity/manim: .github/PULL_REQUEST_TEMPLATE.md

<!--
Thanks for your contribution to ManimCommunity!

Before filling in the details, ensure:
- Your local changes are up-to-date with ManimCommunity/manim
  
- The title of your PR gives a descriptive summary to end-users. Some examples:

>   - `Fixed last animations not running to completion`)
>   - `Added gradient support and documentation for SVG files`)
>   Examples of what *NOT* to do:
>   - `fixed that styling issue` - not descriptive enough
>   - `fixed issue #XYZ` - end-user needs to do further research

Reviewer: The content of this string is actually not markdown, no need for code literals there.

```suggestion
  - Fixed last animations not running to completion
  - Added gradient support and documentation for SVG files
  Examples of what *NOT* to do:
  - "fixed that styling issue" - not descriptive enough
  - "fixed issue #XYZ" - end-user needs to do further research
```

<!--
Thank you for your interest in contributing to ManimCommunity!

> Before filling out the details below, please ensure:
> - Your local changes are up-to-date with ManimCommunity/manim's latest
> changes and any conflicts are resolved.

Author: ```suggestion
Please ensure that your pull request works with the latest
version of ManimCommunity/manim.
```
@jsonvillanueva: I d like to make this first sentence as short as possible, so the chance that people will read the following keyword section is higher.

Reviewer 2: I'd prefer to keep the bullet-point nature of this section but agree it could be more concise. Maybe this should also be part of the checklist in the [# Acknowledgement](https://github.com/ManimCommunity/manim/pull/1073/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R44) heading?

Reviewer 2: After looking at how[ NumPy does it](https://numpy.org/devdocs/dev/development_workflow.html#writing-the-commit-message), we could also have this in our contributing guidelines on the docs since the `# Acknowledgement` section links to it.

Reviewer 2: I've made it shorter, adopted NumPy acronyms, and added the checklist items. Lmk what you think @kolibril13

Author: > I'd prefer to keep the bullet-point nature

I agree, lets do it like you said.

<!--
Thanks for your contribution to ManimCommunity!

Before filling in the details, ensure:
- Your local changes are up-to-date with ManimCommunity/manim

- The title of your PR starts with at least one capitalized acronym:
  API: an API breaking change
  CI: change related to building manim
  BUG: bug fix
  DEP: deprecation, or removal of a deprecated object
  DEV: development tool or utility
  DOC: documentation
  ENH: enhancement or new feature
  MNT: maintenance (refactoring, typos, etc.)
  REV: revert an earlier commit
  STY: style fix (whitespace, PEP8)
  TST: addition or modification of tests

>   REL: manim release

Author: Personally, I don't like these acronyms too much, as it is another learning step as a contributor, and it would take some time to enforce the usage of them.
Additionally,  I think they are a bit redundant with the pr labels.
But in case we don't have a good alternative, I am fine with using them

Reviewer 2: I was just modeling after NumPy... after some quick research into *why* they do this in their repo, it's because they have an [automatic labeler workflow](https://github.com/numpy/numpy/blob/main/.github/workflows/labeler.yml). I agree this can be removed. The [changelog generator script](https://github.com/numpy/numpy/blob/main/tools/changelog.py) NumPy uses doesn't actually take advantage of these acronyms either which seems like a lost opportunity.

## Acknowledgements
- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

> - [ ] The PR title is descriptive enough

Reviewer: The decision whether the PR title is descriptive enough or not is on the reviewer. It's still a good idea to have a checkbox for PR authors as well to make them aware of the issue. Suggestion for slight rewording:

```suggestion
- [ ] I have chosen a descriptive PR title (see top of PR template for examples)
```

Author: Sounds great!
